src/config.py

```diff
@@ -1,20 +1,16 @@
 import os
-from dotenv import load_dotenv
-
-# Load environment variables from the .env file
-load_dotenv()
```

Comment:
My understanding is that you ned this line in here so that the application will read the values in .env into the environment variables

Reply:
Kevin explained that this isn't necessary when running from docker compose